index.html

```diff
@@ -17,7 +18,11 @@
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
   <link rel="stylesheet" href="./CSS/style.css" />
 </head>
+
 <body>
+
+  <!-- Start of Header -->
+
```

Comment:
- [ ] Kindly try to write all reports because you only write the sections in your comments but there are other reports you should try to write, Such as, animations, background images, colours, interaction, multimedia that you did in your project. 
### This is the list of things that you should try to write reports.

1. The report includes a check about page titles.
2. The report includes a check about image text alternatives `(<img alt="alt text ">)`.
3. The report includes a check about text headings.
4. The report includes a check about color contrast.
5. The report includes a check about resize.
6. The report includes a check about interaction.
7. The report includes a check about moving content.
8. The report incldues a check about multimedia.
9. The report includes a check about the basic structure of the page.

- If everything was ok,  make a statement about that (e.g. "Page titles - no issues found").
- If issues were found, link to the corresponding line of code in the Pull Request changes